index.html

```diff
@@ -249,11 +249,49 @@ <h4>Computation steps</h4>
         <li id="step1">Set the <code>root node</code> to the given element, the <code>current node</code> to the <code>root node</code>, and the <code>total accumulated text</code> to the empty string (""). If the <code>root node</code>'s role prohibits naming, return the empty string ("").</li>
         <li id="step2">Compute the text alternative for the <code>current node</code>:
           <ol>
-            <li id="step2A">If the <code>current node</code> is <a class="termref">hidden</a> <strong>and is not</strong> directly referenced by <code>aria-labelledby</code> or <code>aria-describedby</code>, nor directly referenced by a native host language text alternative <a class="termref">element</a> (e.g. <code>label</code> in HTML) or <a class="termref">attribute</a>, return the empty string.
-              <div><details>
-                <summary>Comment:</summary>
-                <p>By default, <a class="termref">assistive technologies</a> do not relay hidden information, but an author can explicitly override that and include hidden text as part of the <a class="termref">accessible name</a> or <a class="termref">accessible description</a> by using <code>aria-labelledby</code> or <code>aria-describedby</code>. </p>
-              </details></div>
+            <li id="step2A">
+            <ol>
+              <li id="step2A.i">If the <code>current node</code> is <strong>not</strong> <a href="https://html.spec.whatwg.org/multipage/rendering.html#being-rendered">being rendered</a> <strong>and is</strong>:
```

Comment:
I clicked on the "not being rendered" link and i'm confused by the definition. And if I as a browser developer am confused, then certainly authors will be as well. Therefore, I think we need specific guidance on each hiding technique so that no one has to guess. Maybe other reviewers will have an idea.

For example, from reading the "not-being-renderered", is it clear with visibility:hidden is rendered or not? It takes up space, unlike visibility:collapse, so that if an <a href>text</a> element is visibility:hidden, there is a visible hole where it would have been. That means it's rendered, correct?

However, I thought that for AccName we treated visibility:hidden the same as display:none.

Also, what about content-visibility? Some content-visibility is rendered, some isn't, depending on the scroll position? Are we saying the label would change depending on whether something is scrolled onscreen or not? It's probably better to just say that content-visiblity:auto content is treated as visible.

My preference would be to simply list the specific hiding techniques that lead to this condition, so there are no questions. But I'm not sure whether other reviewers will balk at having a dependency on another spec.

Reply:
You are right about "not being rendered", I understood it as "a type of hidden that's not aria-hidden", but it's actually describing display:none but not visibility:hidden.

It's probably better to point to the [hidden definition](https://www.w3.org/TR/accname-1.2/#dfn-hidden) and modify it to state clearly to which kinds of hidden we want to consider (as it is now, it might or might not include aria-hidden).

Reply:
This is done, I'm back to reference "hidden" and adding clarification to that.